Client/Frontend/Browser/MailProviders.swift

```diff
@@ -139,3 +139,18 @@ class GoogleGmailIntegration: MailProvider {
         return constructEmailURLString(beginningScheme, metadata: metadata, supportedHeaders: supportedHeaders).asURL
     }
 }
+
+class FastmailIntegration: MailProvider {
+    var beginningScheme = "fastmail://mail/compose?"
+    var supportedHeaders = [
+        "to",
+        "cc",
+        "bcc",
+        "subject",
+        "body"
```

Comment:
I am curious where do these come from? Do mail providers document this support?

Reply:
These are all standards from mailto RFC https://tools.ietf.org/html/rfc6068

Reply:
It appears in the code that different mail providers support different headers, and that RFC doesn't refer to them explicitly. For example, ymail doesn't support `bcc` according to the code. In all likelihood, `bcc` is supported by Fastmail, so I think we can land this unless we find out otherwise.